common/kernel-filter/kp_kernel_filter.cpp

```diff
@@ -126,7 +126,21 @@ extern "C" void kokkosp_init_library(const int loadSeq,
            (filterKernels ? "enabled" : "disabled"));
 
     if (filterKernels) {
-      char* profileLibrary = getenv("KOKKOS_PROFILE_LIBRARY");
+      char* profileLibrary = getenv("KOKKOS_TOOLS_LIBS");
+      if (NULL == profileLibrary) {  // check for backward compatibility with
+                                     // old environment variable
+        printf("KokkosP: KOKKOS_TOOLS_LIBS not set.\n");
+        profileLibrary = getenv("KOKKOS_PROFILE_LIBRARY");
+        if (NULL == profileLibrary) {
+          printf("KokkosP: No library to call in %s\n", profileLibrary);
+          exit(-1);
+        }
+        else {
+         printf("KokkosP: Found that KOKKOS_PROFILE_LIBRARY is set and it will be used.\n");
+         printf("KokkosP: Note that KOKKOS_PROFILE_LIBRARY is a deprecated variable. " 
+         "Please use KOKKOS_TOOLS_LIBS in the future.\n");
+      }  // end check for backward compatability
```

Comment:
Typo compat(i)bility.  I don't see the point of the comment. It is not even the right place.  You are missing a cling brace I don;'t even think this compiles

Reply:
Thanks. I messed it up (in the process of trying to put that comment). See updated version .